Monika After Story/game/script-topics.rpy

```diff
@@ -7789,45 +7795,266 @@ label monika_smoking:
+
+            m 4hubfb "Your loving girlfriend believes in you! I'm sure you can do it!"
+
+        #safeguard for players who tried all options on monika_smoking
+        "I never smoked.":
+            $ persistent._mas_pm_do_smoke = False
+            $ persistent._mas_pm_do_smoke_quit = False
+            $ persistent._mas_pm_do_smoke_quit_succeeded_before = False
+
+            if persistent._mas_pm_do_smoke_lied:
+                $ mas_loseAffection(amount=10)
+                m 2wfc "You said that last time, [player]!"
+                m 2ekc "I just want to see you be the best you can be...{w=0.5}{nw}"
+                m 2dkd "but you keep lying to me."
+                m 2dfc "Just forget it...{w=0.5}"
+                extend 2rfd "I'm not going to ask anymore."
+                $ mas_lockEVL("monika_smoking", "EVE")
+
+            else:
+                $ persistent._mas_pm_do_smoke_lied = True
+                $ mas_loseAffection(reason=13, modifier=3)
+                m 1wud "Wha--"
+                m "But you said..."
+                m 2esc "I clearly remember you telling me you smoked, [player]..."
+                m 4efc "That's no joke, you know? I'm worried about your health!"
+                m 4esc "Was it a mistake when you said that? You wouldn't lie to me on purpose, would you?"
+                m 2dsc "I'll forgive you this time...{w=0.5} {nw}"
+                extend 2esc "but please take me seriously when I ask these questions."
+    return "no_unlock"
+
+label monika_smoking_quit_success:
+    python:
+        persistent._mas_pm_do_smoke_quit_succeeded_before = True
+        persistent._mas_pm_do_smoke_quit = False
+        persistent._mas_pm_do_smoke = False
+
+    m 1sub "Really?! Oh my gosh, I'm so proud of you [player]!"
+    m 2ekbsa "It's such a relief to know you quit smoking! {nw}"
+    extend 2dkbsu "I'll sleep much better at night knowing you're as far as possible from this nightmare."
+    m 4hubfb "Congratulations, my love! I couldn't be more proud to have a [bf] such as you!"
+    m 1rkbfu "Ehehe, if I was here with you I'd treat you to your favorite dish {nw}"
+
+    if mas_globals.time_of_day_3state == "evening":
+        extend "tomorrow."
+    else:
+        extend "tonight."
+
+    m 3hubfb "It's an impressive feat after all! We need to celebrate!"
+    m 3eubsb "Not everyone who wants to quit manages to pull it off."
+    m 1dubfu "You truly are an inspiration, [player]..."
+    m 2eua "Now, I don't want to undermine your victory or anything, {nw}"
+    extend 2euc "but I need you to stay careful from now on."
+    m 4rsc "Many former smokers feel urges to smoke again at some point or another..."
+    m 4wud "You can't give in, not even once! That's how you fall into relapse!"
+    m 2hubfa "But knowing you, you won't let that happen, won't you?"
+    m 2ekbfa "Considering what you've already done, I know you're stronger than this..."
```

Comment:
```suggestion
    m 2ekbfa "Considering what you've already done, I know you're stronger than this~"
```

Reply:
This '~' feels weird to me witth the current expression; unless you want to change it too, I'd propose to keep it to a single point. `m 2ekbfa "Considering what you've already done, I know you're stronger than this."`

Reply:
The exps are likely changing, I haven't reviewed those yet. Though I'm not sure what you mean by doesn't fit, why use `~` all the time with blush exps.

Reply:
It's not about the blush, I just thought of this sentence as an encouragement (liks she believes in you) rather than a tease. If she was teasing I wouldn't see her making this face is all.

Reply:
Ah, so it's that contributing guideline. While a `~` can be used as a tease with the right exp, the vast majority of the time, we use those at the end of like really sweet or encouraging or loving lines, which is what I had in mind.

Reply:
My bad then, didn't realise it could be used that way.